graylog2-web-interface/src/pages/ConfigurationsPage.jsx

```diff
@@ -3,46 +3,55 @@ import PropTypes from 'prop-types';
 import { Col, Row } from 'components/graylog';
 import { DocumentTitle, PageHeader, Spinner } from 'components/common';
 import { PluginStore } from 'graylog-web-plugin/plugin';
-
 import connect from 'stores/connect';
-import CombinedProvider from 'injection/CombinedProvider';
+import StoreProvider from 'injection/StoreProvider';
 
+import ActionsProvider from 'injection/ActionsProvider';
+import PermissionsMixin from 'util/PermissionsMixin';
 import SearchesConfig from 'components/configurations/SearchesConfig';
 import MessageProcessorsConfig from 'components/configurations/MessageProcessorsConfig';
 import SidecarConfig from 'components/configurations/SidecarConfig';
 import EventsConfig from 'components/configurations/EventsConfig';
+import UrlWhiteListConfig from 'components/configurations/UrlWhiteListConfig';
 import DecoratorsConfig from '../components/configurations/DecoratorsConfig';
+import {} from 'components/maps/configurations';
 
 // eslint-disable-next-line import/no-webpack-loader-syntax
 import style from '!style/useable!css!components/configurations/ConfigurationStyles.css';
 
-const { ConfigurationsActions, ConfigurationsStore } = CombinedProvider.get('Configurations');
+const CurrentUserStore = StoreProvider.getStore('CurrentUser');
+const ConfigurationsStore = StoreProvider.getStore('Configurations');
+const ConfigurationsActions = ActionsProvider.getActions('Configuration');
```

Comment:
Using `CombinedProvider` is the preferred way to inject stores and actions and I would keep that, once it was already there.

Reply:
ok I will change that